apps/web/src/routes/TrackHome/TrackHome.jsx

```diff
@@ -68,7 +68,7 @@ function TrackHome () {
   const { setHeaderDetails } = useViewHeader()
   useEffect(() => {
     setHeaderDetails({
-      title: t('Track: {{trackName}}', { trackName: currentTrack?.name || 'loading...' }),
+      title: t('{{trackName}}', { trackName: currentTrack?.name || 'loading...' }),
```

Comment:
Similarly here, if we are just passing the name in and it has no other copy, we should just remove the t function altogether and just directly pass the variable in

Reply:
yeah @brodeur needs to be trained up in the internationalization stuff